vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -369,6 +363,20 @@ void TouchEventHandler::UpdatePointersInViews(
 }
 
 winrt::Microsoft::ReactNative::JSValue TouchEventHandler::GetPointerJson(const ReactPointer &pointer, int64_t target) {
+  // defines button payoad, follows https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button
+  int button = -1;
+  if (pointer.isLeftButton) {
+    button = 0;
+  } else if (pointer.isMiddleButton) {
+    button = 1;
+  } else if (pointer.isRightButton) {
+    button = 2;
+  } else if (pointer.isBarrelButton) {
+    button = 3;
+  } else if (pointer.isHorizontalScrollWheel) {
+    button = 4;
```

Comment:
3/4 don't exactly follow the Web documentations, able to change/add to these if wanted!